src/app/layout/layout.js

```diff
@@ -11,10 +11,14 @@ const Layout = ({ header, main, sidebar, footer, showSidebar, showFooter }) => {
 
     return (
         <div className={wrapperClass}>
-            <header className={css.header}>{header}</header>
+            <header className={`${css.header} hide-for-print`}>{header}</header>
```

Comment:
You can use the `classnames` library in this case.
[Here](https://github.com/dhis2/data-entry-app/blob/TECH-859/print-forms/src/app/layout/layout.js)'s a good example.

Reply:
> You can use the classnames library when applying multiple class names.

updated